pkg/apis/nsx.vmware.com/v1alpha1/addressbinding_types.go

```diff
@@ -0,0 +1,41 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+type AddressBindingSpec struct {
+	// VMName contains the VM's name
+	VMName string `json:"vmName"`
```

Comment:
Question - when later we support address binding for non-VM ports, e.g. PodVM, then we may have to make this one optional; is that an incompatible API change and we may have to update the API version? If yes, I would add omitempty for this one too.

Reply:
Fixed. Thanks.

Reply:
I think it's compatible to change a required field to optional as the change won't break users who use the old client that requires the field. And it doesn't need to increase API version.

Reply:
Ok, in this case, let us keep this field required for now. @gran-vmv

Reply:
Removed omitempty for VMName